test/servico/cst24Test.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import modelo.UsuarioModel;
+
+/*
+ *
+ * @author Mohamad
+ *
+ */
+
+public class cst24Test {
+    
+    public cst24Test() {
+    }
+    
+    @BeforeClass
+    public static void setUpClass() throws NonexistentEntityException, Exception{
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+    
+    @AfterClass
+    public static void tearDownClass() {
+    }
+    
+    @Before
+    public void setUp() throws ParseException, NonexistentEntityException {
+    }
+    
+    @After
+    public void tearDown() {
+    }
+    
+    @Test
+    public void testeEdicaoDeUsuarioCancelada() throws ParseException{
+        
+        //Login como usuário "Administrador" 
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+        
+        //Navegação até "Editar Usuário" 
+        AdministracaoDeUsuariosService editarUs = new AdministracaoDeUsuariosService();
+        List<UsuarioModel> usuarios = editarUs.ListarUsuarios(); 
+        
+        //Seleção e obtenção dos dados do usuário
+        UsuarioModel editarUsuario = usuarios.get(1);
+        assertEquals(editarUsuario.Usuario,"Professor1");
+        assertEquals(editarUsuario.Senha,"123456");
+        
+        try {
+            //Editar usuário
+            editarUs.EditarUsuario("111111", Perfil.PROFESSOR, usuarios.get(1).id);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 83).](https://www.codacy.com/app/calebepb/substituicao/file/2738706050/issues/source?bid=3308597&fileBranchId=3353677#l71)